InTune/InTune/ViewControllers/ExploreViewController.swift

```diff
@@ -15,48 +15,31 @@ class ExploreViewController: UIViewController {
     @IBOutlet private var artistTableView: UITableView!
     @IBOutlet private var featuredArtistCV: UICollectionView!
     
+    let tabsCVDelegate = TagsCVDelegate()
+    let featuredCVDelegate = FeaturedArtistCVDelegate()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        tagsCollectionView.delegate = self
-        tagsCollectionView.dataSource = self
+        tagsCollectionView.delegate = tabsCVDelegate
+        tagsCollectionView.dataSource = tabsCVDelegate
         artistTableView.delegate = self
         artistTableView.dataSource = self
         artistTableView.register(ExploreArtistCell.self, forCellReuseIdentifier: "exploreCell")
-//        featuredArtistCV.register(UINib(nibName: "FeaturedArtist", bundle: nil), forCellWithReuseIdentifier: "featuredArtist")
-//        featuredArtistCV.delegate = self
-//        featuredArtistCV.dataSource = self
+        featuredArtistCV.register(UINib(nibName: "FeaturedArtist", bundle: nil), forCellWithReuseIdentifier: "featuredArtist")
+        featuredArtistCV.delegate = featuredCVDelegate
+        featuredArtistCV.dataSource = featuredCVDelegate
     }
 
 
-}
-
-extension ExploreViewController: UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
-    
-    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        let maxSize: CGSize = UIScreen.main.bounds.size
-        let itemWidth: CGFloat = maxSize.width * 0.20
-        let itemHeight: CGFloat = maxSize.height * 0.20
-        return CGSize(width: itemWidth, height: itemHeight)
-    }
-    
-    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return 10
-    }
-    
-    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "tagCell", for: indexPath) as? TagCell else {
-            fatalError("could not conform to TagCell")
-        }
-        
-        return cell
-    }
-    
-    
 }
 
 
 extension ExploreViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 120
```

Comment:
always create a constant at the top of the file and give logicalName such as defaultHeight = 120 in this for any variable.